_includes/volunteer-signup.html

```diff
@@ -1,23 +1,50 @@
-<form action="//studentrobotics.us12.list-manage.com/subscribe/post?u=1af17c2a3426de258620aae46&amp;id=220e1d8952"
+<!-- Begin MailChimp Signup Form -->
+<div id="mc_embed_signup">
+  <form action="https://studentrobotics.us12.list-manage.com/subscribe/post?u=1af17c2a3426de258620aae46&amp;id=28b16731d1"
       method="post"
       id="mc-embedded-subscribe-form"
       name="mc-embedded-subscribe-form"
-      class="form validate"
-      target="_blank" novalidate>
-  <div id="mc_embed_signup_scroll">
-    <div class="mc-field-group">
-      <label for="mce-EMAIL">Email Address </label>
-      <input type="email" value="" name="EMAIL" class="full-width required email" id="mce-EMAIL" placeholder="Enter your email address">
+      class="form form-left show-labels validate"
+      target="_blank"
+      novalidate>
+    <div id="mc_embed_signup_scroll">
+<div class="mc-field-group">
+	<label for="mce-EMAIL">Email Address </label>
+	<input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL">
+</div>
+<div class="mc-field-group">
+	<label for="mce-FNAME">First Name </label>
+	<input type="text" value="" name="FNAME" class="required" id="mce-FNAME">
+</div>
+<div class="mc-field-group">
+	<label for="mce-LNAME">Last Name </label>
+	<input type="text" value="" name="LNAME" class="required" id="mce-LNAME">
+</div>
+<div class="mc-field-group">
+	<label for="mce-MMERGE3">Location </label>
+	<input type="text" value="" name="MMERGE3" class="" id="mce-MMERGE3">
+</div>
+<div id="mergeRow-gdpr" class="mergeRow gdpr-mergeRow content__gdprBlock mc-field-group">
+    <div class="content__gdpr">
+        <label>Consent</label>
```

Comment:
Do we really need an explicit consent box?

They're giving us their information explicitly so that we can contact them about volunteering. We're not going to _market_ things to them (if we were then they would need to opt in to that), we're only going to tell them about things they're directly asking to be told about.

Reply:
This form was generated using the mailchimp form generator, the consent checkbox can't be turned off, so I suspect their lawyers have looked into this and made this decision

Reply:
Most likely MailChimp will be assuming that we're collecting the data to use it for marketing purposes. We're not. As I understand it, GDPR consent only applies to marketing messages.

Reply:
I'm not certain I agree with you, but I'll remove it anyways because I trust your judgement and really don't want to spend another evening reading GDPR.

Reply:
Is there not an option in the MailChimp generator to say that we're not collecting _marketing_ emails?

Reply:
I think @prophile may have more GDPR understanding than I and thus may have an opinion here.